### PR TITLE
Add users and groups detail to webhooks

### PR DESCRIPTION
**Pull request by PatAltimore**

### CODE REVIEW

File: api-reference/beta/api/subscription_post_subscriptions.md [hunk 1]
```diff
@@ -2,7 +2,20 @@
...
-One of the following **scopes**, depending on the target resource, are required to execute this API: *Mail.Read*, *Calendars.Read*, *Contacts.Read*, *Group.Read.All*, *Files.ReadWrite* or *Files.ReadWrite.All*. ***Note:*** The /beta endpoint allows Application permissions for most resources. Conversations in a Group and OneDrive drive root items are not supported with Application permissions.
+One of the following **scopes**, depending on the target resource, are required to execute this API. Creating a subscription requires read scope to the resource. For example, to get notifications messages, your app needs the `Mail.Read` permission. The following table lists the suggested roles needed for each resource.
+
+|Resource/Item |Scope               |
+|--------------|--------------------|
+| Messages     | Mail.Read          |
+| Calendars    | Calendars.Read     |
+| Contacts     | Contacts.Read      |
+| Files        | Files.ReadWrite    |
+| Users        | User.Read.All      |
```
Comment (SomeshChandra-MSFT) [inline api-reference/beta/api/subscription_post_subscriptions.md]: Last I remember, this permission scope was not sufficient to read users. Will need to check with Jeff/Dan

File: api-reference/beta/resources/webhooks.md [hunk 2]
```diff
@@ -37,18 +38,32 @@ Client must store the subscription ID to correlate a notification with the corre
...
+
+Creating a subscription requires read scope to the resource. For example, to get notifications messages, your app needs the `Mail.Read` permission. The following table lists the suggested roles needed for each resource.
+
+|Resource/Item |Scope               |
+|--------------|--------------------|
+| Messages     | Mail.Read          |
+| Calendars    | Calendars.Read     |
+| Contacts     | Contacts.Read      |
+| Files        | Files.ReadWrite    |
+| Users        | User.Read.All      |
```
Comment (SomeshChandra-MSFT) [inline api-reference/beta/resources/webhooks.md]: same comment as above. The Group.Read.All is sufficient for reading groups, but User.Read.All was not enough for reading users.

**Review by SomeshChandra-MSFT — COMMENTED** [review]
Last I remember, this permission scope was not sufficient to read users. Will need to check with Jeff/Dan

File: api-reference/beta/api/subscription_delete.md [hunk 3]
```diff
@@ -2,7 +2,22 @@
...
+| Calendars    | Calendars.Read     |
+| Contacts     | Contacts.Read      |
+| Files        | Files.ReadWrite    |
+| Users        | User.Read.All      |
+| Groups       | Group.Read.All     |
+| Drive root   | Files.ReadWrite    |
+| Drives associated with Sharepoint sites | Files.ReadWrite.All |
+
+***Note:*** The /beta endpoint allows Application permissions for most resources. Conversations in a Group and OneDrive drive root items are not supported with Application permissions.
+
```
Comment (ghost) [inline api-reference/beta/api/subscription_delete.md]: So there are a few things here (please see email thread for details):
1. We probably won't update Prereq sections in other topics for a few weeks. I'm not sure if you want to lead the charge -- but up to you.
2. We don't want to talk about "roles".
3. Laura shut me down big time on capitalizing "Application" and "Delegated" unless it's at the beginning of a sentence.
4. We somehow lost information about which APIs support consumer scenarios in this -- I can guess based on the permission. I don't think our customers can :).
Maybe we should sort this out in email -- first?

File: api-reference/beta/resources/webhooks.md [hunk 4]
```diff
@@ -35,22 +52,15 @@ Creating a subscription is the first step to start receiving notifications for a
...
-
-You can create a subscription to a specific folder:
-`https://graph.microsoft.com/beta/me/mailfolders('inbox')/messages`
-
-Or to a top-level resource:
-`https://graph.microsoft.com/beta/me/messages`
-
-Or on a drive root item:
-`https://graph.microsoft.com/beta/me/drive/root`
+The following table lists the types of permissions your app can request in order to use webhooks for specific resource types.
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/webhooks.md]: Should we move this lead-in sentence and the table to above the "Create a subscription" section? The same permissions are required to do create, as well as renew and delete.

**Review by jesakowi — APPROVED** [review]
Looks good! Sorry for the long delay.

**Pull request closed without merging**